pkgs/development/libraries/kde-frameworks/kcmutils/series

```diff
@@ -1,2 +1 @@
 kcmutils-follow-symlinks.patch
-kcmutils-debug-module-loader.patch
```

Comment:
This patch is important for us to get useful bug reports. Please leave it in place.

Reply:
If I recall correctly the files this was patching does no longer exists.

Reply:
We need to find out where that code went and apply the patch there. It may even have moved to a different library, unfortunately.

Reply:
I just tried to get things compiling so we know what to look at. But the patches are not very well documented so it's hared to tell why they are needed.

How is this normaly done?